src/set-instance-id-config-files.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This script should run only ONCE.
```

Comment:
hmm, i'm wondering if saving this secret to SSM is a general thing that we want to do in other products as well?  if so we would need to copy/paste all these files into other packer projects which would not be good.

If we are planning to do this in more products (AMIs) then we should consider putting these files in a shared location and reference them in `ec2/sc-ec2-linux-jumpcloud-*.yaml` files in the service-catalog-library repo.

I think it also makes sense to move these files to service-catalog-library so that you can run them with the metadata `command` and pass in ENV VARs (SERVICE_CATALOG_PREFIX, SYNAPSE_ENV_VAR_STRING, KMS_KEY_ALIAS_ENV_VAR_STRING, etc..).  That would prevent it from getting out of sync with definitions in the CFN templates.

Reply:
It depends on if we are planning on using Apache proxy for every ec2 instance. This script is only setting up up the necessary information for our `mod_python` apache script to use for storing to ssm code.

Reply:
split the script into 2 parts and moved over to 
https://github.com/Sage-Bionetworks/service-catalog-utils/pull/18